packages/dds/merge-tree/src/mergeTreeNodes.ts

```diff
@@ -281,6 +281,141 @@ export interface ISegment extends IMergeNodeCommon, Partial<IRemovalInfo>, Parti
 	ack(segmentGroup: SegmentGroup, opArgs: IMergeTreeDeltaOpArgs): boolean;
 }
 
+/**
```

Comment:
nit. could be put these in the perspectives file?

This file was just to split the original merge tree file, and the things here end up exported in the index (mostly).

Reply:
i say the perspectives file, as they all answer questions based on perspective

Reply:
Yeah that makes sense. They were originally in `mergeTree.ts` alongside `wasRemovedAfter`, etc. but I figured they could go alongside `ISegment` because that's the only type they use. Putting them in `perspective.ts` makes sense organizationally, but if the import cycle is a problem that would make it worse.